### PR TITLE
docs(datepicker): add demo for different triggers

### PR DESCRIPTION
**Pull request by ghost**
add demo for different triggers. 
In this pr datepicker should open
 -on hover, 
-on focus(on input) + press any key,
-on double click  

related to #3729

### CODE REVIEW

File: demo/src/app/components/+datepicker/demos/triggers/triggers.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<div class="row">
+  <div class="col-xs-12 col-12 col-sm-6 col-lg-3 form-group">
+    <p>Hover on the input element to open datepicker</p>
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/demos/triggers/triggers.html]: Hover over the input element to open the datepicker

File: demo/src/app/components/+datepicker/demos/triggers/triggers.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <input type="text"
+           class="form-control"
+           placeholder="Datepicker"
+           bsDatepicker
+           triggers="mouseenter:click">
+  </div>
+  <div class="col-xs-12 col-12 col-sm-6 col-lg-3 form-group">
+    <p>Hover on the element and press any button to open datepicker</p>
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/demos/triggers/triggers.html]: Focus on input element and press any key to open the datepicker

File: demo/src/app/components/+datepicker/demos/triggers/triggers.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    <input type="text"
+           class="form-control"
+           placeholder="Daterangepicker"
+           bsDaterangepicker
+           triggers="keydown:click">
+  </div>
+  <div class="col-xs-12 col-12 col-sm-6 col-lg-3 form-group">
+    <p>Сlick twice on the element to open datepicker</p>
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/demos/triggers/triggers.html]: Сlick twice on the element to open the datepicker

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — APPROVED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Review by valorkin — APPROVED** [review]

**Pull request merged**